ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/mainnet/precompiles/privacy/PrivacyPrecompiledContract.java

```diff
@@ -90,7 +88,7 @@ public void setPrivateTransactionProcessor(
 
   @Override
   public Gas gasRequirement(final BytesValue input) {
-    return Gas.of(40_000L); // Not sure
+    return Gas.of(0);
```

Comment:
Is this the EEA specified gas?

Reply:
There is no requirement in the EEA spec. The reasoning here is that implementing EEA compliant private transaction support without a Precompile contract would cost 21000 gas like a normal transaction. This precompile should not consume gas.